src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<Fruit> fruits = new ArrayList<>();

```

Comment:
```suggestions
    public static final Map<String, Integer> fruits = new HashMap<>();
```
simply use string as a key, and the value is going to be the amount of fruit left in storage